cli/command/container/opts.go

```diff
@@ -914,16 +928,24 @@ func parseSecurityOpts(securityOpts []string) ([]string, error) {
 			// "no-new-privileges" is the only option that does not require a value.
 			return securityOpts, errors.Errorf("Invalid --security-opt: %q", opt)
 		}
-		if k == "seccomp" && v != "unconfined" {
-			f, err := os.ReadFile(v)
-			if err != nil {
-				return securityOpts, errors.Errorf("opening seccomp profile (%s) failed: %v", v, err)
-			}
-			b := bytes.NewBuffer(nil)
-			if err := json.Compact(b, f); err != nil {
-				return securityOpts, errors.Errorf("compacting json for seccomp profile (%s) failed: %v", v, err)
+		if k == "seccomp" {
+			switch v {
+			case seccompProfileDefault, seccompProfileUnconfined:
+				// known special names for built-in profiles
+				securityOpts[key] = "seccomp=" + v
```

Comment:
This will be a no-op right?  If so, maybe we can just leave this branch empty?

Reply:
Doh! Yes, you're right. Honestly I got confused, and later on found that `securityOpts[key]` is _NOT_ a map, but an array, and it's directly manipulating the input variable.

I really want to change that, as that feels ... icky (in either case, change `key` to `i` or something!)

I'll remove this line 👍

Reply:
Fixed 👍